base/commands/migration/migration_start.go

```diff
@@ -45,7 +46,13 @@ Selected data structures in the source cluster will be migrated to the target cl
 		}
 	}
 	ec.PrintlnUnnecessary("")
-	sts := NewStages(makeMigrationID(), ec.Args()[0])
+	var updateTopic *hazelcast.Topic
+	sts := NewStartStages(updateTopic, MakeMigrationID(), ec.Args()[0])
+	if !sts.topicListenerID.Default() && sts.updateTopic != nil {
```

Comment:
what is this if for? As I see, it is always true? isnt't it?

Reply:
I think it is better to check them, in the current flow of the program, it may look like always true, but in the future a change in the connect stage (maybe an ignorable error) can break it.